components/service/nimbus/src/test/java/mozilla/components/service/nimbus/ui/NimbusBranchItemViewHolderTest.kt

```diff
@@ -0,0 +1,100 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.nimbus.ui
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.core.view.isVisible
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.support.test.mock
+import mozilla.components.support.test.robolectric.testContext
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.verify
+import org.mozilla.experiments.nimbus.Branch
+import org.mozilla.experiments.nimbus.FeatureConfig
+
+@RunWith(AndroidJUnit4::class)
+class NimbusBranchItemViewHolderTest {
+
+    private val branch = Branch(
+        slug = "control",
+        ratio = 1,
+        feature = FeatureConfig(
+            featureId = "first_switch",
+            enabled = false
+        )
+    )
+    private lateinit var nimbusBranchesDelegate: NimbusBranchesAdapterDelegate
+    private lateinit var selectedIconView: ImageView
+    private lateinit var titleView: TextView
+    private lateinit var summaryView: TextView
+
+    @Before
+    fun setup() {
+        nimbusBranchesDelegate = mock()
+        selectedIconView = mock()
+        titleView = mock()
+        summaryView = mock()
+    }
+
+    @Test
+    fun `sets text on view`() {
+        val view = View(testContext)
+        val holder = NimbusBranchItemViewHolder(
+            view,
+            nimbusBranchesDelegate,
+            selectedIconView,
+            titleView,
+            summaryView,
+            selectedBranch = ""
+        )
+
+        holder.bind(branch)
+
+        verify(selectedIconView).isVisible = false
+        verify(titleView).text = branch.slug
+        verify(summaryView).text = branch.slug
+    }
+
+    @Test
+    fun `sets click listener`() {
```

Comment:
Nit: Lets follow the same naming convention in all tests [Given When Then](https://martinfowler.com/bliki/GivenWhenThen.html).